src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -232,6 +232,22 @@
   <data name="Globals_LaunchModeMaximized.Content" xml:space="preserve">
     <value>Maximized</value>
   </data>
+  <data name="Globals_WindowingBehavior.Header" xml:space="preserve">
+    <value>New windows open in...</value>
+    <comment>This text is a fragment of a sentence that indicates the options below it, in a user interface. It is a header, but it should be read like a sentence that ends with one of {Globals_WindowingBehaviorUseNew.Content}, {Globals_WindowingBehaviorUseAnyExisting.Content}, or {Globals_WindowingBehaviorUseExisting.Content}.</comment>
+  </data>
+  <data name="Globals_WindowingBehavior.HelpText" xml:space="preserve">
+    <value>Controls how new terminal instances attach to existing windows.</value>

```

Comment:
```suggestion
    <value>Controls how new terminal instances launched from wt.exe attach to existing windows.</value>
```

Reply:
This feels a little redundant though, right? Like, _all_ the windows are ones spawned from `wt.exe`. Do we really need to specify that?